ocs_ci/helpers/managed_services.py

```diff
@@ -332,7 +332,7 @@ def verify_storageclient_storageclass_claims(storageclient):
 
     # Wait for the storageclassclaims to be in Ready state
     for sc_claim in sc_claim_objs:
-        for claim_info in TimeoutSampler(timeout=180, sleep=10, func=sc_claim.get):
+        for claim_info in TimeoutSampler(timeout=1200, sleep=10, func=sc_claim.get):
```

Comment:
Please change the `sleep` to `30` to be more reasonable.

Reply:
Thank you, done.